src/common/ApprovalsChip.tsx

```diff
@@ -25,11 +42,75 @@ export default function ApprovalsChip({ approvals }: { approvals: any }) {
     backgroundColor = '#4c8a4c'
   }
 
+  const getRequestResponses = (response) => {
+    let Icon
+    let secondaryText
+
+    if (response === 'Accepted') {
```

Comment:
Instead of using "magic" strings we should use an enum so that we only have to manage the strings in one place. Something like:
```
enum ApprovalStates {
  ACCEPTED = 'Accepted',
  DECLINED = 'Declined',
  'NO_RESPONSE' = 'No Response',
}

if (response === ApprovalStates.ACCEPTED) {
  // ...
}
```